aspnetcore/tutorials/signalr-typescript-webpack.md

```diff
@@ -24,7 +24,22 @@ In this tutorial, you learn how to:
 
 [View or download sample code](https://github.com/aspnet/Docs/tree/master/aspnetcore/tutorials/signalr-typescript-webpack/sample) ([how to download](xref:index#how-to-download-a-sample))
 
-[!INCLUDE [Prerequisites](~/includes/net-core-prereqs-vs-vsc-2.2.md)]
+## Prerequisites
+
+# [Visual Studio](#tab/visual-studio)
+
+* [Visual Studio 2017 version 15.9 or later](https://www.visualstudio.com/downloads/?utm_medium=microsoft&utm_source=docs.microsoft.com&utm_campaign=button+cta&utm_content=download+vs2017) with the **ASP.NET and web development** workload
+* [.NET Core SDK 2.2 or later](https://www.microsoft.com/net/download/all)
+* [Node.js](https://nodejs.org/) with [npm](https://www.npmjs.com/)
```

Comment:
You get npm automatically with node, so this could read (includes [npm](https://www.npmjs.com/))

Reply:
There's a way to exclude it from the installation. Though many folks won't do that, I figured it was best to call out npm as a prereq.

![image](https://user-images.githubusercontent.com/10702007/56599453-317c5680-65bc-11e9-8f5d-426d2f00ef1b.png)